requirements/requirements_nlp.txt

```diff
@@ -8,10 +9,10 @@ gdown
 h5py
 ijson
 jieba
-mamba-ssm==2.2.2; sys_platform == 'linux'
+mamba_ssm @ git+https://github.com/state-spaces/mamba.git@v2.2.2
```

Comment:
does mamba_ssm have a pypi package we can replace this line with?

Reply:
they exist, but installing doesn't really work. We still need to build them, and so far all attempts of building failed because of some missing c-headers. There are many discussions in their project issues around this, and all solutions fall back to git-installs